nessie/src/main/java/org/apache/iceberg/nessie/NessieTableOperations.java

```diff
@@ -104,12 +96,18 @@ protected void doRefresh() {
             content
                 .unwrap(IcebergTable.class)
                 .orElseThrow(
-                    () ->
-                        new IllegalStateException(
+                    () -> {
+                      if (content instanceof IcebergView) {
+                        return new AlreadyExistsException(
+                            "View with same name already exists: %s", key);
```

Comment:
View??

Reply:
While committing the Table, we found that view existing with the same key. Hence we are failing the commit. 

Other catalogs behaves like this. Common testcase in `ViewCatalogTests` will fail If I remove this check.

Reply:
yes this is correct behavior here IMO